artifacts/src/i18n/esignet-i18n-bundle/kn.json

```diff
@@ -120,13 +126,20 @@
   },
   "header": {
     "login_heading": "ಇ-ಸಿಗ್ನೆಟ್ ಮೂಲಕ ಲಾಗಿನ್ ಮಾಡಿ",
+    "login_subheading": "ದಯವಿಟ್ಟು ನಿಮ್ಮ ನೋಂದಾಯಿತ ಬಳಕೆದಾರಹೆಸರು ಮತ್ತು ಪಾಸ್‌ವರ್ಡ್ ಅನ್ನು ನಮೂದಿಸಿ",
     "login_linkName": "ಮುಂಚಿತವಾಗಿ ನೋಂದಾಯಿಸಿ",
     "more_ways_to_sign_in": "ಸೈನ್ ಇನ್ ಮಾಡಲು ಇನ್ನಷ್ಟು ಮಾರ್ಗಗಳು",
     "invalid_auth_factor": "ಅಮಾನ್ಯವಾದ ದೃಢೀಕರಣ ಅಂಶಗಳು ದಯವಿಟ್ಟು ಪುನಃ ಪ್ರಯತ್ನಿಸಿ...",
     "download_now": "ಈಗ ಡೌನ್‌ಲೋಡ್ ಮಾಡಿ",
     "logo_alt": "ಇ-ಸಿಗ್ನೆಟ್",
     "backgroud_image_alt": "ಇ-ಸಿಗ್ನೆಟ್ ಮೂಲಕ ಸೈನ್ ಇನ್ ಮಾಡಿ",
-    "language": "ಭಾಷೆ"
+    "language": "ಭಾಷೆ",
+    "noAccount": "ಖಾತೆ ಇಲ್ಲವೇ?",
+    "signup_with_one_login": "ಏಕೀಕೃತ ಲಾಗಿನ್‌ನೊಂದಿಗೆ ಸೈನ್ ಅಪ್ ಮಾಡಿ"
+  },
+  "footer": {
+    "powered_by": "ನಡೆಸಲ್ಪಡುತ್ತಿದೆ",
+    "logo_alt": "அடிக்குறிப்பு லோகோ"
```

Comment:
Why is the value in tamil here?

Reply:
i will change that

Reply:
make the changes in [c0cf4c9](https://github.com/mosip/artifactory-ref-impl/pull/570/commits/c0cf4c9e23abc33549ca545f46d91cff31d808c4)